report-job-status/action.yml

```diff
@@ -32,6 +39,19 @@ runs:
           ${{ github.event.head_commit.message }}
       shell: bash
 
+    - name: Get job data
+      id: get-job-data
+      shell: bash
+      env:
+        CURL_COMMAND: "curl --location --fail --silent --show-error --retry 5 --retry-delay 5 "
```

Comment:
You can use something like this to make it more readable and fit it to 80 chars:

```
      env:
        CURL_COMMAND: >-
          curl --location
               --fail
               --silent
               --show-error
               --retry 5
               --retry-delay 5
```

Reply:
fixed